app/controllers/application_controller.rb

```diff
@@ -1,10 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :cart
+  helper_method :cart, :current_user
 
   def cart
     @cart ||= Cart.new(session[:cart] ||= Hash.new(0))
   end
 
+  def current_user
```

Comment:
Glad we decided to go ahead with creating this current_user helper! Seems like it will be super useful moving into the authentication stories.